RecFCCeeCalorimeter/src/components/AugmentClustersFCCee.cpp

```diff
@@ -77,22 +87,55 @@ StatusCode AugmentClustersFCCee::initialize()
       showerShapeDecorations.push_back(Form("energy_fraction_%s_layer_%d", detector, layer));
       showerShapeDecorations.push_back(Form("theta_%s_layer_%d", detector, layer));
       showerShapeDecorations.push_back(Form("phi_%s_layer_%d", detector, layer));
+      showerShapeDecorations.push_back(Form("width_theta_%s_layer_%d", detector, layer));
+      showerShapeDecorations.push_back(Form("width_module_%s_layer_%d", detector, layer));
+      showerShapeDecorations.push_back(Form("Ratio_E_max_2ndmax_%s_layer_%d", detector, layer));
+      showerShapeDecorations.push_back(Form("Delta_E_2ndmax_min_%s_layer_%d", detector, layer));
+      showerShapeDecorations.push_back(Form("width_theta_3Bin_%s_layer_%d", detector, layer));
     }
   }
   m_showerShapeHandle.put(showerShapeDecorations);
 
   return StatusCode::SUCCESS;
 }
 
+std::pair<std::vector<int>, std::vector<double>> MergeSumAndSort(std::vector<int>& A, std::vector<double>& B) {
+  std::unordered_map<int, double> elementSum;
+  // traverse vec A, merge the same elements (theta ID) in vec A and sum up the corresponding elements (E_cell) in vec B
```

Comment:
I would move the description of the method above its definition, and also add to the description that the method returns a pair of vectors with the merged cells (unique elements of vector A and sum of values of B for cells with same ID), sorted in ascending order of identifier (explain also why the sorting is needed)